spec/support/shared_context.rb

```diff
@@ -13,3 +13,18 @@
 RSpec.configure do |rspec|
   rspec.include_context "when proper status", proper_status: true
 end
+
+RSpec.shared_context "with ThrottledCall" do
```

Comment:
Since there will be a few more calls that need replacing I extracted this `shared_context` now.